src/overgangsstønad/steg/4-barnasbosted/BarnasBosted.tsx

```diff
@@ -35,48 +21,31 @@ const BarnasBosted: React.FC = () => {
     søknad,
     mellomlagreOvergangsstønad,
     settDokumentasjonsbehovForBarn,
-    settSøknad,
+    oppdaterBarnISoknaden,
   } = useSøknad();
 
-  const barna = søknad.person.barn.filter((barn: IBarn) => {
+  const aktuelleBarn = søknad.person.barn.filter((barn: IBarn) => {
     return !barn.medforelder?.verdi || barn.medforelder?.verdi?.død === false;
   });
 
-  const barnMedDødMedforelder = søknad.person.barn.filter((barn: IBarn) => {
-    return barn.medforelder?.verdi?.død === true;
-  });
-
   const kommerFraOppsummering = kommerFraOppsummeringen(location.state);
   const skalViseKnapper = !kommerFraOppsummering
     ? ESide.visTilbakeNesteAvbrytKnapp
     : ESide.visTilbakeTilOppsummeringKnapp;
 
-  const antallBarnMedForeldre = antallBarnMedForeldreUtfylt(barna);
+  const antallBarnMedForeldre = antallBarnMedForeldreUtfylt(aktuelleBarn);
 
   const [sisteBarnUtfylt, settSisteBarnUtfylt] = useState<boolean>(
-    antallBarnMedForeldre === barna.length
+    antallBarnMedForeldre === aktuelleBarn.length
   );
 
   useMount(() => logSidevisningOvergangsstonad('BarnasBosted'));
 
-  const settBarneliste = (nyBarneListe: IBarn[]) => {
-    settSøknad((prevSoknad: ISøknad) => {
-      return {
-        ...prevSoknad,
-        person: { ...søknad.person, barn: nyBarneListe },
-      };
-    });
-  };
-
-  const [aktivIndex, settAktivIndex] = useState<number>(
-    hentIndexFørsteBarnSomIkkeErUtfylt(barna)
-  );
-
-  const lagtTilBarn = useRef(null);
-
-  const scrollTilLagtTilBarn = () => {
-    setTimeout(() => scrollTilRef(lagtTilBarn), 120);
-  };
+  useEffect(() => {
+    settSisteBarnUtfylt(
+      antallBarnMedForeldreUtfylt(aktuelleBarn) === aktuelleBarn.length
```

Comment:
Kan man bruke `antallBarnMedForeldre` her (linje 36)? Eller vil ikke den være oppdatert?

Reply:
Usikker 😬 Men burde vel være like "up-to-date" som `aktuelleBarn`, kanskje?